aspnetcore/includes/install2.0.md

```diff
@@ -1,4 +1,4 @@
 Install the following:
 
 * [.NET Core 2.0.0 SDK](https://dot.net/core) or later.
-* [Visual Studio](https://www.visualstudio.com/vs/) 15.3 or later.
\ No newline at end of file
+* [Visual Studio](https://www.visualstudio.com/downloads/) 15.3 or later, with the **ASP.NET and web development** and **.NET Core cross-platform development workloads** installed.
```

Comment:
Do we need/want a screenshot? Dan.Roth doesn't like screen shots. He was under the impression we could remove all of that with one install link - for .NET Core 2